nginx/containerbuddy.json

```diff
@@ -1,5 +1,5 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{ .CONSUL }}:8500",
   "onStart": "/opt/containerbuddy/reload.sh",
```

Comment:
This calls https://github.com/tgross/triton-nginx/blob/master/bin/reload.sh#L19, causing it to try `nginx -s reload` before the Nginx starts, triggering an error:

```
2016/03/17 18:53:53 [error] 11366#11366: open() "/var/run/nginx.pid" failed (2: No such file or directory)
nginx: [error] open() "/var/run/nginx.pid" failed (2: No such file or directory)
```